src/host/settings.hpp

```diff
@@ -21,7 +21,7 @@ Revision History:
 #include "../buffer/out/TextAttribute.hpp"
 
 // To prevent invisible windows, set a lower threshold on window alpha channel.
-#define MIN_WINDOW_OPACITY 0x4D // 0x4D is approximately 30% visible/opaque (70% transparent). Valid range is 0x00-0xff.
+constexpr unsigned int MIN_WINDOW_OPACITY = 0x4D; // 0x4D is approximately 30% visible/opaque (70% transparent). Valid range is 0x00-0xff.

```

Comment:
Should this be `WORD`? See that's how much confusion `#define` caused us.

Reply:
And by the way do we prefer `WORD` over `unsigned short`?

Reply:
`WORD` is just a Windows-ism for `unsigned short`. I don't know strictly why other than Windows invented a lot of Windows-isms over the years. We've slowly been moving things to the more standard naming instead of the Windows-isms, so feel free to use `unsigned short` instead.